package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "watch": "tsup --watch",
     "start": "node dist/index.js",
-    "build": "tsup"
+    "build": "tsup",
+    "test:e2e": "pnpm exec playwright test"
```

Comment:
weird you'd need pnpm exec. running `pnpm test:e2e` i would have thought was enough. with `playwright test`